docs/examples/control_ish_lite_ufschem-example.yaml

```diff
@@ -0,0 +1,314 @@
+        
+    mapping: # of _model_ species name to _obs_ species name
+      ish_lite: # specifically for the obs labeled 'ish'
+        temperature_k: "temp"
+        windspeed: "ws"
+        dewpoint: "dew_pt_temp"
+
+    variables:
+      winddir: "wdir"
+      
+    projection: ~
+    plot_kwargs: # optional
+      color: "magenta"
+      marker: "s"
+      linestyle: "-"
+obs:
+  ish_lite: # obs label
+    use_ish_lite: True
+    filename: example:ish-lite:2017-07
+    obs_type: pt_sfc
+    variables: # optional
+      temp:
+        unit_scale: 273.15
+        # ^ optional; Scaling factor
+        unit_scale_method: "+"
+        # ^ optional; Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+        nan_value: -1.0
+        # ^ optional; When loading data, set this value to NaN
+        ylabel_plot: "Temperature (K)"
+        # optional; set ylabel in order to include units and/or other info
+        xlabel_plot: "Temperature (K)"
+        # optional; set ylabel in order to include units and/or other info
+        vmin_plot: 273.15
+        # ^ optional; Min for y-axis during plotting.
+        #   To apply to a plot, change restrict_yaxis = True.
+        vmax_plot: 313
+        # ^ optional; Max for y-axis during plotting.
+        #   To apply to a plot, change restrict_yaxis = True.
+        vdiff_plot: 20.0
+        # ^ optional; +/- range to use in bias plots.
+        #   To apply to a plot, change restrict_yaxis = True.
+        nlevels_plot: 21
+        # ^ optional; number of levels used in colorbar for contourf plot.
+      ws:
+        unit_scale: 1
+        unit_scale_method: "*"
+        nan_value: -1.0
+        ylabel_plot: "Windspeed (m/s)"
+        xlabel_plot: "Windspeed (m/s)"
+        vmin_plot: 0
+        vmax_plot: 15
+        vdiff_plot: 4
+        nlevels_plot: 18
+      wdir: 
+        unit_scale: 1
+        unit_scale_method: "*"
+        nan_value: -1.0
+        ylabel_plot: "Wind Direction (deg)"
+        xlabel_plot: "Wind Direction (deg)"
+        vmin_plot: 0
+        vmax_plot: 370
+        vdiff_plot: 4
+        nlevels_plot: 18
+      dew_pt_temp:
+        unit_scale: 273.15
+        unit_scale_method: "+"
+        nan_value: -1.0
+        ylabel_plot: "Dewpoint (K)"
+        xlabel_plot: "Dewpoint (K)"
+        vmin_plot: 273.15
+        vmax_plot: 313
+        vdiff_plot: 4
+        nlevels_plot: 18
+        
+plots:
+  plot_grp1:
+    type: "timeseries" # plot type
+    fig_kwargs: # optional; to define figure options
+      figsize: [12, 6] # figure size (width, height) in inches
+    default_plot_kwargs:
+      # ^ optional; Define defaults for all plots.
+      #   Important: Model kwargs overwrite these.
+      linewidth: 2.0
+      markersize: 10.
+    text_kwargs: # optional
+      fontsize: 24.
+    domain_type: ["all", "all"]
+    # ^ List of domain types: 'all' or any domain in obs file.
+    #   (e.g., airnow: epa_region, state_name, siteid, etc.)
+    domain_name: ["CONUS", "GLOBAL"]
```

Comment:
For ISH-lite, this is also a global data set, so I would change all of these to all GLOBAL and maybe plot California too.